lib/airbrakex/notifier.ex

```diff
@@ -25,35 +25,38 @@ defmodule Airbrakex.Notifier do
   end
 
   defp add_notifier(payload) do
-    payload |> Dict.put(:notifier, @info)
+    Map.put(payload, :notifier, @info)
```

Comment:
I think it's not needed here, it does not add readability. I tend to avoid needless pipe function calls. Also the guide suggest to avoid needless pipelines:
https://github.com/lexmag/elixir-style-guide#needless-pipeline